SofaKernel/framework/sofa/helper/vector.h

```diff
@@ -184,47 +193,120 @@ class vector<T, CPUMemoryManager<T> > : public std::vector<T, std::allocator<T>
     void fastResize(size_type n) {
         this->resize(n);
     }
+
 };
 
 
+
+
+// ======================  operations on standard vectors
+
+// -----------------------------------------------------------
+//
+/*! @name vector class-related methods
+
+*/
+//
+// -----------------------------------------------------------
+//@{
+/** Remove the first occurence of a given value.
+
+The remaining values are shifted.
+*/
+template<class T1, class T2>
+void remove( T1& v, const T2& elem )
```

Comment:
If I well understand, this function was already existing, but you reimplement it and move it in the file?
Moving a function in a file makes the review really complicated.